post_office/lockfile.py

```diff
@@ -44,7 +44,7 @@ def __init__(self, lock_filename, timeout=None, force=False):
     def get_lock_pid(self):
         try:
             return int(open(self.lock_filename).read())
-        except IOError:
+        except OSError:
```

Comment:
Any reason why this is changed?

Reply:
In Python 3.3+ this exceptions are now simple alias. OSError is actually thrown